roles/user_verify_present/tasks/main.yml

```diff
@@ -8,3 +8,6 @@
 
 - name: Verify users in /etc/passwd
   command: "grep atomic-user-{{ uvp_uid }}:x:{{ uvp_uid }}:{{ uvp_uid }}::/home/atomic-user-{{ uvp_uid }}:/bin/bash /etc/passwd"
+
+- name: Verify password is a SHA-512 hash
+  command: "grep '^atomic-user-{{ uvp_uid }}:$6.*' /etc/shadow"
```

Comment:
Yeah, that would be correct.  Guess I should have read `man crypt` more carefully.